<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="windowTitle">
   <string>Preferencias</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <!-- Row 0: Carpeta de datos -->
     <item row="0" column="0">
      <widget class="QLabel" name="lblDataDir">
       <property name="text">
        <string>Carpeta de datos</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="layoutDataDir">
       <item>
        <widget class="QLineEdit" name="txtDataDir">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBrowseDataDir">
         <property name="text">
          <string>Cambiar…</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>

     <!-- Row 1: Perfil -->
     <item row="1" column="0">
      <widget class="QLabel" name="lblProfile">
       <property name="text">
        <string>Perfil</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="layoutProfile">
       <item>
        <widget class="QComboBox" name="cmbProfile"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnNewProfile">
         <property name="text">
          <string>Nuevo perfil…</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>

     <!-- Row 2: Tema -->
     <item row="2" column="0">
      <widget class="QLabel" name="lblTheme">
       <property name="text">
        <string>Tema</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cmbTheme"/>
     </item>
    </layout>
   </item>

   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
